### PR TITLE
Add code for CPX Pinata

### PR DESCRIPTION
**Pull request by TheKitty**

### CODE REVIEW

File: Cirecuit_Playground_Express_Pinata/code.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Circuit Playground Express Piñata by Dano Wall for Adafruit Industries
+# CircuitPython code by Mike Barela for Adafruit Industries, MIT License
+import time
+import gc
+import random
+import board
+import pulseio
+from adafruit_motor import servo
+from adafruit_circuitplayground.express import cpx
+gc.collect()
```
Comment (ladyada) [inline Cirecuit_Playground_Express_Pinata/code.py]: i dont believe this is necessary anymore, as of 4.x we collect after each import

Comment (TheKitty) [reply Cirecuit_Playground_Express_Pinata/code.py]: It failed to work before I added it. Shall I try again?

Comment (ladyada) [reply Cirecuit_Playground_Express_Pinata/code.py]: what cpy version are you running?

Comment (TheKitty) [reply Cirecuit_Playground_Express_Pinata/code.py]: 3.x I think on CPX

Comment (ladyada) [reply Cirecuit_Playground_Express_Pinata/code.py]: try 4.x

Comment (TheKitty) [reply Cirecuit_Playground_Express_Pinata/code.py]: 4.0.0 works w/o garbage collection. Tested

**Comment by ladyada** [comment]
sweet, we solve a lot of memory issues in 4x so please have the guide tell em to update to 4+

**Pull request merged**